modules/fybrik-implicit-copy-batch/Makefile

```diff
@@ -3,7 +3,7 @@ all: helm-all
 
 ROOT_DIR := ../..
 DOCKER_NAME := fybrik-implicit-copy-batch
-DOCKER_TAGNAME := 0.1.0
+DOCKER_TAGNAME ?= 0.1.0
```

Comment:
should we use the default tag as 0.0.0 ? as in the fybrik case

Reply:
Sure, I can try.

Reply:
but you use `0.1.0` instead of `master`